src/compiler/compile_rust_code.ts

```diff
@@ -18,47 +20,47 @@ function compileRustCodeWithPodman(
     canisterName: string,
     stdio: IOType
 ) {
-    execSync(
+    execSyncPretty(
         `podman exec ${dockerContainerName} rm -rf /.azle/${canisterName}`,
-        { stdio }
+        stdio
```

Comment:
Here to remind my self to understand why this is just stdio and not an object

Reply:
Okay so if I wanted to do "['inherit', 'pipe', 'inherit']" I would just pass that in instead of doing {stdio: ['inherit', 'pipe', 'inherit']}. Is that right?

Reply:
No it wouldn't do that, I'll leave a comment on the actual implementation